apps/nextjs-website/src/components/Header.tsx

```diff
@@ -11,23 +11,33 @@ import {
   MegaHeaderData,
   HeaderSublink,
 } from '@/lib/fetch/header';
+import { Locale } from '@/lib/fetch/siteWideSEO';
+
+const LinkLabelValues = {
+  it: 'NOVITÀ',
+  en: 'NEW',
+  de: 'NEU',
+  fr: 'NOUVEAU',
+  sl: 'NOVO',
+};
 
 const makeSublink = (
   sublink: HeaderSublink,
-): {
-  label: string;
-  href: string;
-} => ({
+  locale: Locale,
+): { label: string; href: string; badge?: string } => ({
   label: sublink.label,
   href: sublink.page.data
     ? sublink.page.data.attributes.slug +
       (sublink.sectionID ? `#${sublink.sectionID}` : '')
     : (sublink.externalURL ?? ''),
+  ...(sublink.isNew && { label: LinkLabelValues[locale] }),
```

Comment:
```suggestion
  ...(sublink.isNew && { badge: LinkLabelValues[locale] }),
```
I didn't see that "label" is already used. That's my bad, sorry.